core/src/filter/resources/jenkins/slaves/remoting-info.properties

```diff
@@ -0,0 +1,6 @@
+# Remoting version, which is embedded into the core
+# This version MAY differ from what is really classloaded (see the "Pluggable Core Components", JENKINS-41196)
+remoting.embedded.version=${remoting.version}
```

Comment:
Or this could be computed from `remoting*.jar!/META-INF/MANIFEST.MF#Version` using `Which`.

Reply:
I would prefer to keep it as is. 
 
* At some point we may want to write a "Remoting Compatibility Tester" tool or so
* Not sure how which is going to behave once we have pluggable core components in place